CONTRIBUTING.md

```diff
@@ -0,0 +1,137 @@
+# Thank you for considering contributing!
+
+Contributions are always welcome. We accept [bug fixes](#bugs), [proposals and
+new ideas](#proposals-for-new-content), or simply editorial improvements.
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement).
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/abi/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/abi/pulls).
+
+## Proposals for new content
+
+We are open to any proposal that will improve either the content or the
+presentation of the ABI.
+
+Arm recommends that you
+present them in a [pull
+request](https://github.com/ARM-software/acle/pulls), along with the
+following details:
+
+1. A rationale in support of the changes.
+2. A design document to keep track of the reasoning that made the
+   proposal reach its current state.
+
+Please note that this extra information is not a requirement for
+submitting new content. Contributors are trusted to use their judgment
+to decide whether or not the proposal needs this information. Arm
+recommends that you add this information so that it is easier for new
+ideas to be discussed and possibly accepted, especially for changes of
+great impact.
+
+Instead of a pull request, you can also create an
+[issue](https://github.com/ARM-software/abi/issues), in which you describe your
+proposed change. However beware that opening an issue will have less chance to
+make it into the ABI, as someone will need to do the leg-work to make actual
+changes.
+
+If you want to make ABI changes that for some reason can't be discussed in
```

Comment:
If you want to make ABI changes that for some reason can't be discussed in

change 'some' to 'any'